app.py

```diff
@@ -8,7 +8,7 @@ def chat(user_input):
     try:
         response = message_to_bot(user_input, clf, learn_response)
     except:
-        return jsonify({'message': ('Unable to get response', 0)}, 500)
+        return jsonify({'message': ('Unable to get response', learn_response)}, 500)
     
     return jsonify({'message': response}, 200)
```

Comment:
`learn_response` needs to be passed here too

Reply:
response already has `learn_response` as `message_to_bot` function returns message as well as `learn_response`. Kindly check @vishakha-lall

Reply:
Thanks for clarifying this!